build/build-extensions.ps1

```diff
@@ -118,6 +127,108 @@ function CleanOutput([string] $rootPath) {
     Write-Host "  Current size: $(GetFolderSizeInMb $rootPath) Mb"
 }
 
+function CreatePatchedSiteExtension([string] $siteExtensionPath) {
+  try {
+    Write-Host "SiteExtensionPath is $siteExtensionPath"
+    $officialSiteExtensionPath = "$siteExtensionPath\$extensionVersionNoSuffix"
+    $baseVersion = "$majorMinorVersion.0"
+    
+    # Try to download base version
+    New-Item -Itemtype "directory" -path "BaseZipDirectory" -Force > $null
```

Comment:
Can you change all of these new paths to be `$buildOutput\BaseZipDirectory`, etc? That way no matter where you run this script from, we've got the right path being created.

Reply:
`$buildOutput` is directory that has artifacts for output right ? `$zipOutput` below does use a subdirectory from `$buildOutput`... `$BaseZipDirectory` is just a temp one that shouldn't get uploaded as part of publish

Reply:
It is -- but only a subset of the folders get published under buildoutput. (i.e. /packages, /siteextension, /symbols, etc. There are other temp directories created in there that don't get published as well. I'd rather we do everything in that one directory (b/c we clean it up at the start of the script) just so things don't get left dangling elsewhere.

Reply:
Done.